modules/build/src/main/scala/scala/build/options/ConfigMonoid.scala

```diff
@@ -81,17 +82,23 @@ object ConfigMonoid {
       main ++ defaults
   }
 
-  implicit def map[K, V](implicit valueMonoid: ConfigMonoid[V]): ConfigMonoid[Map[K, V]] =
```

Comment:
This is definitely fishy. Some config values need maps to be "summed" the way they were before.